src/euphorie/content/browser/survey.py

```diff
@@ -448,3 +448,18 @@ def _filename(self):
             mapping=dict(title=self.context.title),
         )
         return f"{translate(filename, context=self.request)}.docx"
+
+
+class FindSolutionTitleDuplications(BrowserView):
+    """"""
+
+    @property
+    def measures(self):
+        brains = api.content.find(
+            portal_type="euphorie.solution",
+            path="/".join(self.context.getPhysicalPath()),
+        )
+        for brain in brains:
+            obj = brain.getObject()
+            if obj.description in obj.action:
```

Comment:
shouldn't it be the other way around? assuming that `obj.action` is the title...

```suggestion
            if obj.action in obj.description:
```

Reply:
No, this is an oira weirdness. What the ticket calls the title is in the `description` attribute and what it calls the description is actually the `action` attribute.